tests/core/pyspec/eth2spec/test/altair/block_processing/sync_aggregate/test_process_sync_aggregate.py

```diff
@@ -480,6 +480,55 @@ def test_proposer_in_committee_without_participation(spec, state):
         raise AssertionError("failed to find a proposer in the sync committee set; check test setup")
 
 
+@with_altair_and_later
+@spec_state_test
+@always_bls
+@with_presets([MINIMAL], reason="prefer short search to find matching proposer")
+def test_proposer_in_committee_without_participation__zero_pre_balance(spec, state):
+    committee_indices = compute_committee_indices(state, state.current_sync_committee)
+
+    # NOTE: seem to reliably be getting a matching proposer in the first epoch w/ ``MINIMAL`` preset.
+    for _ in range(spec.SLOTS_PER_EPOCH):
+        block = build_empty_block_for_next_slot(spec, state)
+        proposer_index = block.proposer_index
+        proposer_pubkey = state.validators[proposer_index].pubkey
+        proposer_is_in_sync_committee = proposer_pubkey in state.current_sync_committee.pubkeys
+
+        # Need at least one participant to have a valid sync committee signature
+        if proposer_is_in_sync_committee:
+            participant_index = (committee_indices.index(proposer_index) + 1) % len(committee_indices)
+        else:
+            participant_index = 0
+
+        validator_index = committee_indices[participant_index]
+        participation = [False for _ in committee_indices]
+        participation[participant_index] = True
+        participants = [validator_index]
```

Comment:
I'm not sure if this approach triggers the condition. If I understand you code, there is exactly one committee participant, which is the proposer. The proposer starts with 0 balance, and gets rewarded for participating in the sync committee, but there's no balance decrease for the propose.

To trigger the condition we need to (1) have the non-participation penalty of the proposer index saturate to 0, and have a later participant reward increase the proposer balance to > 0. So we should set the proposer participation to False and at least one bit to True whose bit index is _greater_ than the proposer location in the sync committee.

I